pkgs/tools/system/9base/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib
+, stdenv
+, fetchgit
+, pkg-config
+, patches ? [ ]
+, enableStatic ? stdenv.hostPlatform.isStatic
+, buildPackages
+}:
+
+stdenv.mkDerivation {
+  pname = "9base";
+  version = "unstable-2019-09-11";
+
+  src = fetchgit {
+    url = "https://git.suckless.org/9base";
+    rev = "63916da7bd6d73d9a405ce83fc4ca34845667cce";
+    hash = "sha256-CNK7Ycmcl5vkmtA5VKwKxGZz8AoIG1JH/LTKoYmWSBI=";
+  };
+
+  patches = [
+    ./config-substitutions.patch
+    ./dont-strip.patch
+    # plan9port dropped their own getcallerpc implementations
+    # in favour of using gcc/clang's macros or a stub
+    # we can do this here too to extend platform support
+    # https://github.com/9fans/plan9port/commit/540caa5873bcc3bc2a0e1896119f5b53a0e8e630
+    # https://github.com/9fans/plan9port/commit/323e1a8fac276f008e6d5146a83cbc88edeabc87
+    ./getcallerpc-use-macro-or-stub.patch
+  ] ++ patches;
+
+  preBuild = ''
+    substituteInPlace config.mk \
+      --replace "@PREFIX@" "${placeholder "out"}" \
+      --replace "@LDFLAGS@" "${if enableStatic then "-static" else ""}" \
+      --replace "@CC@" "${stdenv.cc.targetPrefix}cc" \
+      --replace "@AR@" "${stdenv.cc.targetPrefix}ar"
+  '' +
+  lib.optionalString (stdenv.hostPlatform != stdenv.buildPlatform) ''
```

Comment:
```suggestion
  lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```

Reply:
I think this suggestion is missing a `!`, added with that amendment